torchvision/io/__init__.py

```diff
@@ -17,11 +17,15 @@
     write_video,
 )
 from .image import (
-    read_image,
     decode_image,
+    decode_jpeg,
+    decode_png,
```

Comment:
I would consider `decode_jpeg` and `decode_png` as less public methods and was not planning on exposing them right now, but let's move forward with it and expose everything